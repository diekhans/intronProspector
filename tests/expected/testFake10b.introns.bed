chr_fake	34666	35513	sj0	1	-
chr_fake	130577	131433	sj1	1	+
chr_fake	131550	132801	sj2	1	+
chr_fake	132879	133326	sj3	1	+
chr_fake	133449	136100	sj4	1	+
chr_fake	136650	139594	sj5	1	+
chr_fake	426491	489097	sj6	1	+
chr_fake	487560	488264	sj7	37	+
chr_fake	488522	488947	sj8	45	+
chr_fake	489118	490086	sj9	48	+
