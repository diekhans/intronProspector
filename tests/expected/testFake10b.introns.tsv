chrom	intron_start	intron_end	strand	uniq_mapped_count	multi_mapped_count	unsure_mapped_count	max_left_overhang	max_right_overhang
chr_fake	34666	35513	-	1	0	0	81	20
chr_fake	130577	131433	+	1	0	0	72	29
chr_fake	131550	132801	+	1	0	0	85	16
chr_fake	132879	133326	+	1	0	0	80	17
chr_fake	133449	136100	+	1	0	0	15	85
chr_fake	136650	139594	+	1	0	0	75	26
chr_fake	426491	489097	+	0	0	1	9	21
chr_fake	487560	488264	+	37	0	0	68	90
chr_fake	488522	488947	+	44	0	1	87	91
chr_fake	489118	490086	+	41	0	7	93	90
